compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -234,10 +232,11 @@ class Context(val contextId: Context.ContextId, languageVersion: LanguageVersion
             )
           )
         )
-      case Failure(e: SError) => handleFailure(e)
+      case Failure(e: Error) => handleFailure(e)
       case Failure(e: ScriptF.FailedCmd) =>
         e.cause match {
-          case e: SError => handleFailure(e)
+          case e: Error => handleFailure(e)
+          case e: speedy.SError.SError => handleFailure(Error.RunnerException(e))
```

Comment:
Is this needed for backward compatibility?

Reply:
`FailCmd` may still contain some `SError`    -- I am not completley sure where they come from.
Refactoring `FailCmd` is not completly trivial as it may handle any `Throwable` but is definitively not part of this PR.

Reply:
the errors in `FailedCmd` should come from `IdeLedgerClient`.

Reply:
@cocreature I will double check.

Reply:
@cocreature `//compiler/damlc/tests:script-service` is failing when I remove the line 238.

Reply:
thx will take a look separately, let’s leave it out of this PR.

Reply:
nvm this is definitly expected. As discussed this morning the use of exceptions at least in some of those places is probably somewhat questionable but at least in the current state this makes sense.